src/Components/TopBar.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+export default function TopBar() {
+  return (
+    <section className="jumbotron text-center">
+      <h1 className="jumbotron-heading">Go Fund Mo</h1>
```

Comment:
Go Fund Mo? Is it Mo or Me?
(There used to be an instructor here called Mo so I don't know if this is a legacy inside joke)

Reply:
Haha I assumed this was a legacy inside joke! But if he's no longer kicking around I'll revert back to Me. 😄

Reply:
```suggestion
      <h1 className="jumbotron-heading">Go Fund Me</h1>
```